hw/top_earlgrey/data/chip_testplan.hjson

```diff
@@ -379,6 +379,9 @@
             sleep the peripheral can continue its signaling even in deep sleep. The testbench
             verifies the correctness of the reflected values once the chip goes into deep sleep.
             This is replicated for DIO pins as well.
+
+            In this test, passthrough feature is not tested. The feature is
+            covered in other tests such as chip_sw_sleep_pwm_pulses.
```

Comment:
could we explicitly x-ref the tests that satisfy the passthrough functions?
@msfschaffner and i only found two modules where this needs to be done.. `pwm` and `sysrst_ctrl`.